run_tests.sh

```diff
@@ -1,75 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I like the cleanup of the run_tests.sh/run-tests.sh. We should have only one way to do it (TBD), and apply everywhere. Want help there?